ci-operator/config/openshift/prometheus/openshift-prometheus-master.yaml

```diff
@@ -49,7 +49,7 @@ tests:
     from: src
 - as: react-app-test
   commands: |
-    nodejs_version=v16.13.1
+    nodejs_version=v20.5.1
```

Comment:
(nit) `v20.14.0` seems to be the latest LTS release: https://nodejs.org/en/download/package-manager, but not a blocker.

Reply:
Prometheus upstream installs v20.5.1 which is why I picked it.